src/components/NotFound/NotFound.js

```diff
@@ -1,11 +1,17 @@
+import { Link } from "react-router-dom";
+
 function NotFound() {
     return (
         <div className="NotFound">
             <h1>404</h1>
             <p>Uh-oh... seems like you’re lost. We couldn’t find that page.</p>
 
             <div>
-                <button className="primary">take me home scotty!</button>
+                <button className="primary">
+                    <Link to="/" onClick={() => {}}>
```

Comment:
you can remove the onClick property if it's empty (you don't need it in this case)